modules/search/instant-search/components/search-sidebar.scss

```
@@ -1,8 +1,7 @@
 .jetpack-instant-search__sidebar {
 	.jetpack-filters.widget_search,
 	.jetpack-instant-search-wrapper {
-		// TODO: To be updated to break-medium
-		@media ( max-width: #{ ( $break-small ) } ) {
+		@media ( max-width: #{ ( $break-medium ) } ) {
```

We might actually need the following:
```suggestion
		@media ( max-width: #{ ( $break-medium + 1 ) } ) {
```

---

Great catch! I just noticed by browser window resizes on every two pixels, so it skipped this exact resolution. Fixed in https://github.com/Automattic/jetpack/pull/14423/commits/f506922bed3d370443e3c14bef73f4d3e54071cb